packages/components/src/Table/Cells/Cell.css

```diff
@@ -1,4 +1,6 @@
 .cell {
+  padding: var(--horizontal-inset) calc(var(--horizontal-inset) / 2)
```

Comment:
Maintains even spacing between cells (in the event of a numeric and non-numeric cell being in sequence, the headers and data would bump up against each other)

Other way of handling this could be simply setting one end to have 0 and the other to have full, instead of dividing by 2?